### PR TITLE
Fix flaky test for Safari

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
Fix flaky test for Safari

FYI @rohitvinnakota @MykolaGalian @johngwilkinson @NicholasHallman

### CODE REVIEW

File: karma.sauce.conf.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ const customLaunchers = {
 module.exports = config => {
 	config.set(
 		merge(createDefaultConfig(config), {
+			// from https://support.saucelabs.com/hc/en-us/articles/225104707-Karma-Tests-Disconnect-Particularly-When-Running-Tests-on-Safari
+			// to avoid DISCONNECTED messages
+			browserDisconnectTimeout: 10000, // default 2000
+			browserDisconnectTolerance: 1, // default 0
+			browserNoActivityTimeout: 4 * 60 * 1000, //default 10000
+			captureTimeout: 4 * 60 * 1000, //default 60000
```
Comment (anhill-D2L) [inline karma.sauce.conf.js:36]: Thank you for the comments and link 👍

**Review by anhill-D2L — APPROVED** [review]

File: karma.sauce.conf.js (hunk 1 above)
Comment (Vitalii-Misechko) [reply karma.sauce.conf.js:36]: Oh.. this shouldn't be here. It does not work AFAIK. I copied it from `karma.conf.js` in hope it will fix our timeouts.

Comment (anhill-D2L) [reply karma.sauce.conf.js]: Is it to fix a different issue from the flakey timeouts we've been seeing lately?

Comment (Vitalii-Misechko) [reply karma.sauce.conf.js:36]: When I started worked on this flaky-test I could not figure out the route cause of `sends timings with a performance event`-test. So, I tried to fix timeouts that you mentioned. Now, when I reverted these lines I see that introducing them had some positive effect. Thus, I'm reverting my revert ))

**Review by anhill-D2L — APPROVED** [review]

File: test/model/telemetry-helper.test.js [hunk 2]
```diff
@@ -36,11 +36,19 @@ describe('TelemetryHelper', () => {
...
 
-			console.dir(fetchSandbox.lastCall());
 			const lastCall = fetchSandbox.lastCall();
 
 			expect(lastCall[0]).to.equal('http://example.com/');
-			const event = JSON.parse(await lastCall[1].body);
+
+			const body = await lastCall[1].body;
+
+			if (body === '[object ReadableStream]') {
```
Comment (bemailloux) [inline test/model/telemetry-helper.test.js:45]: If this is a `ReadableStream`, can we just read the stream to get the contents?

Comment (Vitalii-Misechko) [reply test/model/telemetry-helper.test.js:45]: Unfortunately, It's not a `ReadableStream` it's string with value that equals to`[object ReadableStream]` which is returned in body when `await lastCall[1].body`-promise is resolved

**Pull request merged**